src/recipes/conditionalMultiline.ts

```diff
@@ -0,0 +1,32 @@
+import { defineRecipe } from '@pandacss/dev'
+
+const conditionalMultilineRecipe = defineRecipe({
```

Comment:
I'm not clear why you have `conditionalMultilineRecipe` and `multilineRecipe`. Can you explain?

Reply:
Not deleting multilineRecipe originally was a mistake, but I've changed to using it with base styles like you said below.